src/backend/controllers/pages.ts

```diff
@@ -203,6 +259,16 @@ class Pages {
       throw new Error('Please, fill page Header');
     }
   }
+
+  /**
+   * Check if the given page is root
+   *
+   * @param {page} page - page to check
+   * @returns {boolean}
+   */
+  private static isRootPage(page:Page):boolean {
```

Comment:
It would better to move it to the model and call like `page.isRoot`

```js
class Page {
  get isRoot(){
    return this._parent === '0';
  }
}
```